recipes/arraymap/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "arraymap" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 16edc1b39cb66d62c6e2561feb236371f7819fcbe5225200d1a4663cc79568a3
+
+build:
+  number: 0
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy ==1.18.5  # [py==37]
+    - numpy ==1.18.5  # [py==38]
+    - numpy >=1.18.5  # [py>=39]
```

Don't constrain this so it uses the global pinning.
```suggestion
    - numpy
```